src/main/java/com/synopsys/integration/detect/workflow/componentlocationanalysis/GenerateComponentLocationAnalysisOperation.java

```diff
@@ -0,0 +1,49 @@
+package com.synopsys.integration.detect.workflow.componentlocationanalysis;
+
+import com.synopsys.integration.blackduck.api.generated.view.DeveloperScansScanView;
+import com.synopsys.integration.detect.configuration.DetectUserFriendlyException;
+import com.synopsys.integration.detect.configuration.enumeration.ExitCodeType;
+import com.synopsys.integration.detect.workflow.bdio.BdioResult;
+import com.synopsys.integration.detect.workflow.file.DetectFileUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class GenerateComponentLocationAnalysisOperation {
+    private static final String CLL_INPUT_FILE_NAME = "components-source.json";
```

Comment:
'CLL_' prefix is not needed as we have no other input and output files in this class.